test/Connect.js

```diff
@@ -150,23 +150,25 @@ describe('Connect', () => {
       expect(uport.network.rpcUrl, 'uport.network.rpcUrl').to.equal(provider.provider.host)
     })
 
-    it('returns provider which calls connect.requestAddress on getCoinbase', (done) => {
-      const uport = new Connect('test app')
-      const mnid = '2oeXufHGDpU51bfKBsZDdu7Je9weJ3r7sVG'
-      const addressTest = '0x122bd1a75ae8c741f7e2ab0a28bd30b8dbb1a67e'
-      const verifyResponse = sinon.stub().callsFake((jwt) => Promise.resolve({'name': 'uPort Demo', '@type': 'App', 'description': 'Demo App', 'url': 'demo.uport.me', 'address': '0x122bd1a75ae8c741f7e2ab0a28bd30b8dbb1a67e'}))
-      uport.verifyResponse = verifyResponse
-      uport.requestAddress = sinon.stub()
-      const web3 = new Web3(uport.getProvider())
-      web3.eth.getCoinbase((error, address) => {
-        expect(address).to.equal(addressTest)
-        done()
-      })
-      // Fake response
-      const resId = 'addressReqProvider'
-      const res = { id: resId, res: resJWT, data: '' }
-      uport.pubResponse(res)
-    })
+    // it('returns provider which calls connect.requestAddress on getCoinbase', (done) => {
```

Comment:
Is there a reason for this being commented out?

Reply:
So connect.requestAddress was removed in a previous PR, but somehow this test kept passing until now and I forgot to go back and figure it out.  I'll figure out what the issue was and push a working test